### ISSUE TITLE
Secured Layers Process Convoluted

### ISSUE THREAD

**Issue by mudnmaps** [doug-m/CollectorAuroraTesting#173]
### Issue
four step process now for dual authentication when it used to be two.

### Expected Behavior
Used to be we logged into Collector, and then we logged into a map to access the data.

### Map and Credentials
Hard to replicate this due to having things on our internal servers and requiring DOT authentication.
* Group Name:
* Map Name:

### Steps to Reproduce
Log into collector,
Then get message that says secured layers - you need to sign in to acces some of the layers in this map view (put a LINK to sign in here... but no only choice is Cancel or View)
Choose View
Then data loads with error triangles
Then click on error triangle and finally get prompted to log in to the data
Then the data still doesn't load until you click on the layer in the list.

### Device Information
* Device:  IPad Air2
* OS Version:  iOS 11.4.1

![aurora1](https://user-images.githubusercontent.com/19308681/44553069-fdc9f380-a6f1-11e8-8172-68b8e855982d.png)
![aurora2](https://user-images.githubusercontent.com/19308681/44553070-fdc9f380-a6f1-11e8-8a77-079c410dc42b.png)
![aurora3](https://user-images.githubusercontent.com/19308681/44553071-fdc9f380-a6f1-11e8-9ce5-db98cf2f87a6.png)
![aurora4](https://user-images.githubusercontent.com/19308681/44553072-fdc9f380-a6f1-11e8-9e5b-eda062f6b9a4.png)

**Comment by Kevin-GIS** [comment]
Hi @mudnmaps 

Thanks for this feedback.

Are all the layers in the map from the same service or separate services?

Could you please elaborate on the part where you said "Then the data still doesn't load until you click on the layer in the list."? After you've provided the credentials after initially tapping on a layer in the Layers list, the layer doesn't become visible in the map and load? You then need to tap on the same layer again after that for it to load in the map?

**Comment by mudnmaps** [comment]
They are all from the same service.  Let me try again and see if it behaves the same way I remember.
